src/tests/3-corax+.8o

```diff
@@ -247,14 +276,38 @@
   #test Fx33
   y += 5
   drawop imF im3
+  
+  # >= 100
   i := scratchpad
   v6 := 137
   bcd v6
   load v2
   i := image-ok
-  if v0 != 1 then i := image-no
-  if v1 != 3 then i := image-no
-  if v2 != 7 then i := image-no
+  if v0 != 1 begin i := image-no jump skipOtherFx33Tests
+  if v1 != 3 begin i := image-no jump skipOtherFx33Tests
+  if v2 != 7 begin i := image-no jump skipOtherFx33Tests
+  
+  # < 100
+  i := scratchpad
+  v6 := 65
+  bcd v6
+  load v2
+  i := image-ok
+  if v0 != 0 begin i := image-no jump skipOtherFx33Tests
+  if v1 != 6 begin i := image-no jump skipOtherFx33Tests
+  if v2 != 5 begin i := image-no jump skipOtherFx33Tests
+  
+  # < 10
+  i := scratchpad
+  v6 := 4
+  bcd v6
+  load v2
+  i := image-ok
+  if v0 != 0 begin i := image-no jump skipOtherFx33Tests
+  if v1 != 0 begin i := image-no jump skipOtherFx33Tests
+  if v2 != 4 then i := image-no
+  
+  : skipOtherFx33Tests
```

Comment:
Labels go in column one (no indentation), except where we're doing self-modifying code magic or other strange things that aren't really labels in the traditional sense. Also, I would use a more descriptive name, like `fx33-failed`. Same goes for `after0E` above.